src/utils.php

```diff
@@ -32,3 +32,19 @@ function pathToUri(string $filepath): string {
     $filepath = implode('/', array_map('urlencode', explode('/', $filepath)));
     return 'file:///' . $filepath;
 }
+
+/**
+ * Transforms URI into file path
+ * 
+ * @param string $uri
+ * @return string
+ */
+function uriToPath(string $uri)
+{
+    $position = strpos($uri, '://');
+    if (! $position) {
+        return $uri;
+    }
+    $path = substr($uri, $position + 3);
+    return strpos($path, ':') ? str_replace('/', "\\", substr($path, 1)) : $path;
```

Comment:
Not that happy with this implementation.
- If possible I would like to use [`parse_url`](http://php.net/manual/en/function.parse-url.php)
- Needs to handle %-encoding
- Needs to replace `/` with `DIRECTORY_SEPERATOR`